TCF2 docs

bretg opened the pull request:
@MartianTribe - @jaiminpanchal27 suggested you might be interested in reviewing the TCF2 docs?

jeanstemp commented on dev-docs/modules/gdprEnforcement.md:
```diff
@@ -0,0 +1,201 @@
+| TCF Purpose | In-Scope Activity | Enforcement Activity | Optional Controls |
+| --- | --- | --- | --- |
+| Purpose 1 - Store and/or access information on a device | usersync pixels | May result in preventing one or more usersync activities for one or more vendors. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
+| Purpose 1 - Store and/or access information on a device | user ID modules | May result in preventing one or more UserID modules to not activate. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
```
Can we get rid of the double negative here (prevent...to not)? Either "May prevent one or more UserID modules from activating" or "May allow one or more UserID modules to activate". (I wasn't sure which was correct.)

jeanstemp commented on dev-docs/modules/gdprEnforcement.md:
```diff
@@ -0,0 +1,201 @@
+| --- | --- | --- | --- |
+| Purpose 1 - Store and/or access information on a device | usersync pixels | May result in preventing one or more usersync activities for one or more vendors. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
+| Purpose 1 - Store and/or access information on a device | user ID modules | May result in preventing one or more UserID modules to not activate. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
+| Purpose 1 - Store and/or access information on a device | device storage | May result in preventing one or more adapters or modules not being able to read or write cookies or localstorage in the user's browser. | Do not enforce Purpose 1. Do not enforce Purpose 1 vendor signals. Do not enforce Purpose 1 for vendor V. |
```
Please fix the double negative here, it's confusing (preventing...not being able to). Preventing not being able to would be allowing, but not sure that's what's meant.

jeanstemp commented on dev-docs/modules/gdprEnforcement.md:
```diff
@@ -0,0 +1,201 @@
+| gdpr.rules[].purpose | `String` | The only currently supported value is "storage", corresponding to TCF Purpose 1. | "storage" |
+| gdpr.rules[].enforcePurpose | `Boolean` | Determines whether to enforce the purpose consent or not. The default in Prebid.js 3.x is not to enforce purposes. The plan for Prebid.js 4.0 is to enforce consent for Purpose 1 and no others. | true |
+| gdpr.rules[].enforceVendor | `Boolean` | Determines whether to enforce vendor signals for this purpose or not. The default in Prebid.js 3.x is not to enforce vendor signals. The plan for Prebid.js 4.0 to enforce signals for Purpose 1 and no others. | true |
+| gdpr.rules[].vendorExceptions | `Array of Strings` | Defines which biddercodes or module names do not fall under the enforcement of either enforcePurpose=true or enforceVendors=true. Publishers will not be able to validate that any vendor in this list is enforcing the appropriate GDPR rules. | true |
```
Went with Steve's suggestion here.

bretg commented:
Thanks both.

> Publishers will not be able to validate that any vendor in this list is enforcing the appropriate GDPR rules.

I like what I had because it's clear and direct, but the compromise is to remove the statement altogether. and just leave the fact and not the interpretation. I've changed it to "Defines a list of biddercodes or module names that are exempt from the enforcement of this Purpose."

Also fixed the double negatives.

bretg commented:
Feature released with 3.14 -- merging docs
